### PR TITLE
Updated guide/security-best-practices

### PR DESCRIPTION
**Pull request by SilverFire**
A little bit more information about the CSRF

### CODE REVIEW

File: docs/guide/security-best-practices.md [hunk 1]
```diff
@@ -139,10 +139,9 @@ from a user browser are made by the user himself. It could be false.
...
-requesting an image or requesting a page so when user opens a page with such `img` tag he's being logged out from
-`an.example.com`.
+requesting an image or requesting a page so when user opens a page with such `img` tag, the browser will send the GET request to that URL, and the user will be logged out from `an.example.com`. 
 
-That's the basic idea. One can say that logging user out is nothing serious. Well, sending POST isn't much trickier.
+That's the basic idea. One can say that logging user out is nothing serious, but bad guys can do much more, using this idea. Imagine that some website has a URL `http://an.example.com/purse/transfer?to=anotherUser&amout=2000`, visiting which with a GET request, causes transfer of $2000 from authorized user account to user `anotherUser`. We know, that browser will always send GET request to load an image, so we can modify code to accept only POST requests on that URL. Unfortunately, this will not save us, because bad guys can put some JavaScript code instead of `<img>` tag, which allows them to send POST requests on that URL.
```
Comment (klimov-paul) [inline docs/guide/security-best-practices.md]: "bad guys" is not a proper term for the guide content.

Comment (SilverFire) [inline docs/guide/security-best-practices.md]: but it's used in the previous subparagraph

"everything is OK but one day bad guys are somehow posting"...

Comment (samdark) [inline docs/guide/security-best-practices.md]: Makes the guide a bit informal but I think that's OK once or twice. This way it's easier to read it.

Comment (samdark) [inline docs/guide/security-best-practices.md]: In this case though, I'd use "attacker".

Comment (samdark) [inline docs/guide/security-best-practices.md]: `a url` → `an url`.

Comment (samdark) [inline docs/guide/security-best-practices.md]: `, visiting which with a GET request` → `. Accessing it using GET request`

Comment (SilverFire) [inline docs/guide/security-best-practices.md]: @samdark, 

> `a url` → `an url`

Found [this](http://itknowledgeexchange.techtarget.com/writing-for-business/which-is-correct-a-url-or-an-url/). What do you think?

Comment (samdark) [inline docs/guide/security-best-practices.md]: Interesting :)

**Comment by samdark** [comment]
Thanks!

**Pull request merged**